### ISSUE TITLE
Feature request: Button to automatically assign shortcuts

### ISSUE THREAD

**Issue by d4rky-pl** [jiripospisil/idea-gototabs#5]
I know that README states:

>

Which is why I'd be rad if instead of it happening automatically there was a button / buttons that allowed you to set the shortcuts to `cmd/alt/ctrl + [0-9]` (or combinations).

The reason is that I'm installing this on every JetBrains IDE I'm using and I'm also trying to convince others to install it (because it provides soooo much better experience) but setting the keymap every time takes a bit of time :)

**Comment by jiripospisil** [comment]
I can see that being helpful, maybe even having it as another item in the menu (Window -> Go to...). PRs welcome :)

In the meantime, you can export your settings including keyboard shortcuts ([Exporting and Importing Settings](https://www.jetbrains.com/help/idea/exporting-and-importing-settings.html)) and import it to other IntelliJ IDEs.

**Comment by d4rky-pl** [comment]
Yeah, but this exports all keymaps and I don't have those 100% the same mappings in all IntelliJ apps :)
